unix/lib/time.mli

```diff
@@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
 
 (** Timeout operations. *)
 
-val sleep : float -> unit Lwt.t
```

Comment:
Why is this being removed? It's OK to add a new function if we need to, but removing the old one serves no purpose.